### ISSUE TITLE
NTVDMx64 interferes with OTVDM

### ISSUE THREAD

**Issue by HomerBender77** [leecher1337/ntvdmx64#37]
When I install NTVDMx64, the OTVDM stops working (OTVDM shows up in Task manager as a background process for a few seconds and then closes). What can I do?

By the way, OTVDM is another Github project by otya128 which is designed for running 16-bit Windows programs on 64/32-bit Windows.

**Comment by leecher1337** [comment]
There was an issue with OTVDM that was fixed in https://github.com/leecher1337/ntvdmx64/commit/38b45c54ad08d2bfd23cd9bbaa7ebcb4f20d035d
However this was due to the fact that the loader redirected 16bit apps to NTVDMx64 which was disabled in this release (#define WOW16_SUPPORT was removed).
As you say that OTVDM starts, I wonder which issues make OTVDM close then and how this is all related to NTVDMx64. Does it also occur if you start OTVDM manually with the 16bit app as a parameter?
If you try to use OTVDM with HAXM support, the issues might be related to OTVDM, as I didn't get to work HAXM support in OTVDM either, so HAXM shouldn't be enabled in OTVDM (but it's not enabled in default OTVDM installation anyway)
I tried with most recent OTVDM on Windows 7 (Version 6.1 Build 7601: Service Pack 1) and I didn't find any issues, NTVDMx64 and OTVDM work nicely together and in fact I recommend this combination myself for 16bit Support on Win64.
